source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,65 @@ Sensitive Data Encryption
+* `Encryption at rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_
+
+Another MongoDB feature that prevents unauthorized access of data is `Client-Side Field Level Encryption (CSFLE) <https://docs.mongodb.com/manual/core/security-client-side-encryption/>`_. This feature allows a developer to selectively encrypt individual fields of a document on the client-side before it is sent to the server. This keeps the encrypted data private from the providers hosting the database as well as any user that has direct access to the database.
+
+This guide provides steps for setup and implementation of CSFLE.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
 
 Problem
 ~~~~~~~
 
-Pellentesque non mi et est congue mattis pretium sit amet mi. Praesent
-nec tortor nec arcu molestie dapibus. Phasellus at nunc ut quam molestie
-finibus. Suspendisse finibus lectus eget nulla eleifend, vel consequat
-leo rhoncus. Phasellus lacinia tortor id aliquet condimentum. Morbi
-sodales est a leo rhoncus, non pretium neque dignissim.
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, appointments, and vital sign measurement reports for a fictitious company, *MedcoMD*. None of the patient data is public, and certain data such as their social security number (SSN) and vital sign measurements are particularly sensitive. It is important for the company and the patient that the data is kept private and secure.
+
+MedcoMD needs this system to satisfy the following use cases:
+
+- Doctors use the system to access Patients' medical records and add new vital sign measurements.
+- Receptionists use the system to schedule appointments for Patients, who are identified by a combination of contact information and their SSN.
+- Receptionists cannot access nor update Patients' medical records.
+
+MedcoMD is also concerned with disclosure of private information through any of the following methods:
+
+- Printing the information or leaving it displayed on a screen where unauthorized people can view it
+- Communicating on an insecure network
+- Gaining access to the database on one of the permissioned users
+
+What can MedcoMD do to balance the functionality and access restrictions of their Medical Care Management System?
 
 Solution
 ~~~~~~~~
 
-Praesent sit amet elit pretium mi molestie porta a sit amet ipsum. Sed
-sollicitudin purus et mi blandit sodales.
+The MedcoMD software engineers review the Medical Care Management System specification and determine that Doctors and Receptionists require different levels of access to the data.
+
 
-- Deterministic vs Non-Deterministic
+While role-based access control could separate the access, the granularity is only as fine as a collection
+
+
+In order to minimize the chance of disclosure of sensitive data, they decide encryption is the best option
```

Comment:
nit: These first three sentences could be one single paragraph.

Reply:
Personal notes. Replaced with other arguments for CSFLE